splitgraph/cloud/models.py

```diff
@@ -259,6 +259,7 @@ class AddExternalRepositoryRequest(BaseModel):
     credential_data: Optional[Dict[str, Any]]
     schedule: Optional[IngestionSchedule]
     initial_private: bool = False
+    tunnel: Optional[bool] = False
```

Comment:
Can this be bool, default to false, instead of an Optional? LGTM otherwise.

Reply:
I started with that and got a typecheck error if I'm not mistaken.